pymodbus/client/asynchronous/factory/udp.py

```diff
@@ -68,7 +67,7 @@ def async_io_factory(host="127.0.0.1", port=Defaults.Port, framer=None,
     from pymodbus.client.asynchronous.async_io import init_udp_client
     loop = kwargs.get("loop") or asyncio.get_event_loop()
     proto_cls = kwargs.get("proto_cls", None)
-    cor = init_udp_client(proto_cls, loop, host, port)
+    cor = init_udp_client(proto_cls, loop, host, port, **kwargs)
```

Comment:
before this, remove from kwargs (i.e. {arg:argv[arg] for arg in kwargs if arg not in ["proto_cls","loop","host","port"]}